bin/common

```diff
@@ -34,6 +31,16 @@ Please check the log file for more details: $LOG_URL/$(basename $LOG)"
   fi
 }
 
+measure() {
+  args="$@"
+  measure-and-output "dotty-bench-bootstrapped/jmh:run $args -d $PROG_HOME/out -nowarn"
+}
+
+measure-run() {
+  args="$@"
+  measure-and-output "dotty-bench-run/jmh:run $args"
```

Comment:
@brunnerant Have you checked locally that the function `measure` can parse the runtime test results?

Reply:
Yes, I did it, and it works. In fact, since the output also comes from JMH, I could reuse part of the script.

Reply:
Thanks for confirmation.